contrib/inventory/apstra_aos.py

```diff
@@ -0,0 +1,398 @@
+#!/usr/bin/env python
+#
+# (c) 2017 Apstra Inc, <community@apstra.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Apstra AOS external inventory script
+=================================
+
+Ansible has a feature where instead of reading from /etc/ansible/hosts
+as a text file, it can query external programs to obtain the list
+of hosts, groups the hosts are in, and even variables to assign to each host.
+
+To use this:
+ - copy this file over /etc/ansible/hosts and chmod +x the file.
+ - Copy both files (.py and .ini) in your prefered directory
+
+More information about Ansible Dynamic Inventory here
+http://unix.stackexchange.com/questions/205479/in-ansible-dynamic-inventory-json-can-i-render-hostvars-based-on-the-hostname
+
+Tested with Apstra AOS 1.1
+
+This script has been inspired by the cobbler.py inventory. thanks
+"""
+
+import argparse
+import ConfigParser
```

Comment:
May want to use the configparser in 'ansible.compat.six.moves.configparser' here. Otherwise there are py2/py3 compat issues.